book/website/community-handbook/consistency/consistency-structure.md

```diff
@@ -0,0 +1,106 @@
+(ch-consistency-structure)=
+# Structure
+
+Structure refers to how _The Turing Way_ is organised.
+A good structural framework ensures that _The Turing Way's_ content is presented in an orderly manner.
+
+(ch-consistency-structure-hr)=
+## Hard Requirements
+
+The hard checks that deal with _The Turing Way's_ structure include: 
+
+(ch-consistency-structure-hr-emptyfiles)=
+### Check 1: Do not add any empty files in the `_toc.yml`
+
+The `_toc.yml` file is where _The Turing Way's_ table of contents (ToC) lives.
+Some files referenced in the ToC are empty, and their inclusion means that readers are able to navigate to them while reading _The Turing Way_.
+This negatively impacts the reader's experience as they go through the book.
+
+```{figure} ../../figures/empty_toc_file.png
+---
+name: empty_toc_file
+alt: An empty file that was included in the Turing Way's Table of Contents. Readers can still navigate to empty files when they are included in the table of contents.
+---
+Empty files included Turing Way's Table of Contents can still be accessed by readers.
+```
+
+A general suggestion is to remove references to such files from the ToC and raise an issue in _The Turing Way_ Github [repo](https://github.com/alan-turing-institute/the-turing-way) for content to be written for those files. 
+When the content is written, the files can then be added to the ToC.
+
+For example, [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/1391) curates a list of empty and incomplete files currently in _The Turing Way_, and 
+[this PR](https://github.com/alan-turing-institute/the-turing-way/pull/1448) writes content for the `Data Licences` subchapter shown in the {ref}`image <empty_toc_file>`.
+
+(ch-consistency-structure-hr-structure)=
+### Check 2: Ensure chapters follow a consistent structure
+
+_The Turing Way_ is made up of five guides that contain several chapters and subchapters. 
+However, these chapters do not follow a uniform structure, taking away from the reader's experience.
+
+```{figure} ../../figures/recommended_chapter_structure.png
+---
+name: recommended_chapter_structure
+alt: The recommended structure for chapters in The Turing Way. Chapters should have a landing page with a sections for prerequisites, summary, and why the chapter is useful. Chapters should also have a checklist and a resources subchapter in addition to the rest of the chapter content.
+---
+Recommended structure for chapters in The Turing Way.
+```
+
+The recommended structure for chapters is shown in the {ref}`<recommended_chapter_structure>`.
+The landing page of the chapter should include information about any prerequisites required to understand the chapter, a summary of the chapter's content, and an explanation of why the chapter is useful.
+While a chapter can contain as many or as little subchapters as necessary, it should also contain a separate checklist and resources subchapter.
```

Comment:
```suggestion
- **Subchapters:** A chapter can contain any number of subchapters or none at all. However, please note the next requirement about the checklist and resources subchapters.
```

Reply:
Not sure if this makes it clearer, or more confusing (in that it's redundant information).